libres/lib/analysis/ies/ies_enkf_data.cpp

```diff
@@ -221,45 +202,26 @@ void ies_enkf_data_augment_initialE(ies_enkf_data_type *data,
                 m++;
             }
         }
-
-        if (dbg) {
-            int m_nrobs = std::min(obs_size_msk - 1, 50);
-            int m_ens_size = std::min(ens_size_msk - 1, 16);
-            matrix_pretty_fprint_submat(data->E, "data->E", "%11.5f",
-                                        data->log_fp, 0, m_nrobs, 0,
-                                        m_ens_size);
-        }
     }
 }
 
 void ies_enkf_data_store_initialA(ies_enkf_data_type *data,
                                   const matrix_type *A) {
     if (!data->A0) {
         // We store the initial ensemble to use it in final update equation                     (Line 11)
-        bool dbg = ies_enkf_config_get_ies_debug(data->config);
         int m_state_size = std::min(matrix_get_rows(A) - 1, 50);
```

Comment:
with `dbg` removed, `m_state_size` and `m_ens_size` is unused.

Reply:
Ok